crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -40,6 +47,8 @@ def restore_chef
         begin
           [:nodes, :roles, :clients, :databags].each do |type|
             Dir.glob(@data.join("knife", type.to_s, "**", "*")).each do |file|
+              # skip client "crowbar"
+              next if type == :clients && file.basename.to_s =~ /^crowbar.json$/
```

Comment:
```file``` is not a Pathname here yet.

Just moving the next line in front of this one should help